controllers/kmsissuer_controller.go

```diff
@@ -191,7 +200,8 @@ func (r *KMSIssuerReconciler) manageSuccess(ctx context.Context, log logr.Logger
 }
 
 // manageFailure
-func (r *KMSIssuerReconciler) manageFailure(ctx context.Context, log logr.Logger, issuer *kmsiapi.KMSIssuer, issue error, message string) error {
+func (r *KMSIssuerReconciler) manageFailure(ctx context.Context, issuer *kmsiapi.KMSIssuer, issue error, message string) error {
+	log := r.Log.WithValues("name", issuer.Name, "namespace", issuer.Namespace)
```

Comment:
Is there any difference in performance when logging is disabled between:
```go
log := r.Log.WithValues("name", issuer.Name, "namespace", issuer.Namespace)
log.Error(issue, message)
```
and
```go
r.Log.WithValues("name", issuer.Name, "namespace", issuer.Namespace).Error(issue, message)
```

Reply:
No difference in performance.

`WithValues` doesn't not log anything. It just adds a named key-value pairs to following calls to log.Info/Error 
https://github.com/go-logr/logr/blob/master/logr.go#L47-L50